### DISCUSSION TITLE
Blocking pageview fire in iframe, possible?

### DISCUSSION THREAD

**Discussion by HFulcher** [plausible/analytics#2436] [Anything Else]
I have a bit of javascript that I use to display an embedded view of a webpage when hovering over a link ([see here](https://github.com/HFulcher/huwfulcher.com/blob/master/_includes/link-previews.html)). The hover over functionality takes the html of the link including the `<head>` which contains the plausible tracking script. This has the effect of triggering a page view for that embedded view (understandably). I'm trying to think of a way to stop that from happening, I don't want to consider a hover over as a pageview. Is there a way to nullify the trigger or better yet, remove the tag from the iframe on load?

**Comment by metmarkosaric** [comment]
hi @HFulcher! perhaps something like our manual script extension could be of use to you for this situation? see https://plausible.io/docs/script-extensions#scriptmanualjs

**Reply by HFulcher** [reply]
Neat! I'll take a look, thanks
